doc/gpo/README.md

```diff
@@ -0,0 +1,22 @@
+# Group Policy Objects
+
+Since version 0.64, PowerToys is released on GitHub with GroupPolicyObject files. You can check these releases on https://github.com/microsoft/PowerToys/releases .
+
+## How to install
+
+- Copy "PowerToys.admx" to "C:\Windows\PolicyDefinitions" or equivalent path if Windows is installed in another path.
+- Copy "PowerToys.adml" to "C:\Windows\PolicyDefinitions\en-US" or equivalent path if Windows is installed in another path. If you have a different language installed, copy into your language folder as well.
+
+The "Local Group Policy Editor" (gpedit.msc) will now show PowerToys policies under "Local Computer Policy" > "Computer Configuration" > "Administrative Templates" > "Windows Components" > "Microsoft PowerToys".
+
+## Policies
```

Comment:
```suggestion
## Policies

### Scope
Currently we only support policies in the computer scoop.

### Policy path
You will find the policies under Computer/Administrative Templates/Microsoft PowerToys
```

Reply:
I've added this information on a different commit, to reflect more recent changes.